### ISSUE TITLE
Update installation instructions now that React Native supports autolinking

### ISSUE THREAD

**Issue by pcowgill** [tradle/react-native-crypto#43]
Now that React Native supports autolinking, will the installation instructions for this package change at all?

https://github.com/react-native-community/cli/blob/master/docs/autolinking.md

Thanks!

**Comment by mvayngrib** [comment]
@pcowgill yes, for RN >= 0.60, `react-native link react-native-randombytes` is no longer necessary
